.github/ISSUE_TEMPLATE/issue report.yml

```
@@ -4,52 +4,111 @@ title: "[Issue]: "
 labels: [bug, triage]
 body:
   - type: markdown
+    id: introduction
     attributes:
       value: |
-        Thanks for taking the time to report an issue. Before submitting a report, please do the following:
-        1. Please head to our forum or chat rooms and troubleshoot with volunteers if you haven't already. Links can be found here: https://jellyfin.org/contact/
-        2. Please search the bug tracker for similar issues. If you do find one, please comment there instead of opening a new bug report.
-        3. If you decide to open a new report, please provide as much detail as possible.
-        4. Please **ONLY** report **ONE** issue per report. If you are experiencing multiple issues, please open multiple reports.
+        ### Thank you for taking the time to report an issue!
+        Please keep in mind that Jellyfin is a [free and open-source](https://jellyfin.org/docs/general/about) project, made up entirely and exclusively of **volunteers** who donate their free time to the project.
+  - type: checkboxes
+    id: before-posting
+    attributes:
+      label: "This issue respects the following points:"
+      description: All conditions are **required**. Failure to comply with any of these conditions may cause your issue to be closed without comment.
+      options:
+        - label: This is a **bug**, not a question or a configuration issue; Please visit our forum or chat rooms first to troubleshoot with volunteers, before creating a report. The links can be found [here](https://jellyfin.org/contact/).
+          required: true
+        - label: This issue is **not** already reported on [GitHub](https://github.com/jellyfin/jellyfin/issues?q=is%3Aissue+is%3Aopen+label%3Abug) _(I've searched it)_.
+          required: true
+        - label: I'm using an up to date version of Jellyfin Server stable, unstable or master; We generally do not support previous older versions. If possible, please update to the latest version before opening an issue.
+          required: true
+        - label: I agree to follow Jellyfin's [Code of Conduct](https://jellyfin.org/docs/general/community-standards.html#code-of-conduct).
+          required: true
+        - label: This report addresses only a single issue; If you encounter multiple issues, kindly create separate reports for each one.
+          required: true
+  - type: markdown
+    id: preliminary-information
+    attributes:
+      value: |
+        ### General preliminary information
+
+        We like ask you to keep the following things in mind when creating this issue:
+
+        1. Fill in as much of the template as possible. When you are unsure about the relevancy of a section, do include the information requested in that section. Only leave out information in sections when you are completely sure about it not being relevant.
+        2. Provide as much details as possible. Do not assume other people to know what is going on.
```

```suggestion
        2. Provide as much detail as possible. Do not assume other people to know what is going on.
```
I "think" you made a confusion between the singular and plural of a noun

---

No problem I'm not native English speaker else just like to learn the language it self

---

You are correct. I asked around and will change it to singular. Thank you!